### ISSUE TITLE
Use overlays for alternative helm installs

### ISSUE THREAD

**Issue by psschwei** [Qiskit-Extensions/quantum-serverless#584]
### What is the expected enhancement?

Currently, we have two values files: a generic one and one specific for IBM Cloud that are essentially identical (save a few IBM Cloud specific bits in the latter). To prevent drift (and to ease the burden on maintainers), we should migrate to using an overlay (helmfile / kustomize / somethign else) for environment-specific config options.

**Comment by Tansito** [comment]
I totally agree @psschwei . My idea, but let me know your opinion, is to start investigating this as soon as we start with the helm configuration for our deployment internally. And It will depends on if we finally can move out the infrastructure configuration to another repository, something that I want to attack sooner than later (cc: @pacomf )

**Comment by Tansito** [comment]
Well, not depends on but I think it will simplify our lifes if we have our helm in another repository.

**Comment by psschwei** [comment]
Agree on the separation of infrastructure 

xref #625

**Comment by psschwei** [comment]
I think at this point we could just move https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/charts/quantum-serverless/values-ibm.yaml to the internal repo and close this one out (?)

**Comment by Tansito** [comment]
Yes, I would remove it directly from this repository. It's not needed anymore.

**Comment by psschwei** [comment]
The underlying issue was resolved in #1015, so don't think this is needed anymore
